core/templates/mkadmin.html

```diff
@@ -7,14 +7,17 @@
   </form>
 
   <ul id="admin_list">
+    <% var x = 0 %>
     <% allAdminsFromDB.rows.forEach(function(admin) { %>
       <li><form method=post>
         <%= admin.netid %>
+        <%= namesFromDB[x] %>
```

Comment:
The forEach callback signature includes as a second argument the index of the current item. You could remove llines 10 and 20 and instead do something like this and have the same effect:
`<% allAdminsFromDB.rows.forEach(function(admin, x) { %>`